src/handlers/list-targets.handler.ts

```diff
@@ -5,20 +5,43 @@ import {
     parseTargetStatus
 } from '../utils';
 import { Logger } from '../logger.service/logger';
-import { EnvironmentDetails } from '../http.service/http.service.types';
+import { EnvironmentDetails, TargetUser, VerbType } from '../http.service/http.service.types';
 import { cleanExit } from './clean-exit.handler';
-import { SsmTargetStatus, TargetSummary, TargetType } from '../types';
-import { map, includes, uniq } from 'lodash';
+import { ClusterSummary, TargetStatus, TargetSummary, TargetType } from '../types';
+import { includes, map, uniq } from 'lodash';
+import { PolicyQueryService } from '../http.service/http.service';
+import { ConfigService } from '../config.service/config.service';
 
 
 export async function listTargetsHandler(
+    configService: ConfigService,
     logger: Logger,
     argv: any,
     dynamicConfigs: Promise<TargetSummary[]>,
     ssmTargets: Promise<TargetSummary[]>,
+    clusters: Promise<ClusterSummary[]>,
     envsPassed: Promise<EnvironmentDetails[]>) {
-    // await and concatenate
+
+    const clusterTargets = (await clusters).map<TargetSummary>((cluster, _index, _array) => {
```

Comment:
We aren't using `_index` and `_array`:

```suggestion
    const clusterTargets = (await clusters).map<TargetSummary>((cluster) => {
```